lib/AST/ASTDumper.cpp

```diff
@@ -2625,11 +2625,11 @@ class PrintExpr : public ExprVisitor<PrintExpr> {
     printCommon(E, name) << ' ';
     if (auto checkedCast = dyn_cast<CheckedCastExpr>(E))
       OS << getCheckedCastKindName(checkedCast->getCastKind()) << ' ';
-    OS << "writtenType='";
-    if (GetTypeOfTypeRepr)
-      GetTypeOfTypeRepr(E->getCastTypeRepr()).print(OS);
-    else
-      E->getCastType().print(OS);
+        OS << "writtenType='";
```

Comment:
This is incorrect code alignment.

Reply:
Whoops thanks. Should have run formatted again after the last commit -- will fix